src/open_inwoner/components/templates/components/Form/FileInput.html

```diff
@@ -3,7 +3,7 @@
 <div class="form__control file-input">
     {% render_card direction="vertical" %}
         {% icon icon="upload" icon_position="before" outlined=True %}
-        <input class="file-input__input" id="{{ field.auto_id }}" name="file" type="file"{% if field.field.required %} required{% endif %}>
+        <input class="file-input__input" id="{{ field.auto_id }}" name="file" type="file"{% if field.field.required %} required{% endif %} multiple>
```

Comment:
Should this be dynamic based on the field type, and shouldn't we (dynamically) change the labels into plurals accordingly?

Reply:
`multiple` should be set dynamically and has been fixed. I'm not sure about the labels. There's no change in the backend if the client drops a file or two into the widget, so I'm not sure what could decide if the label should be singular or plural. Passing that info front to back seems overkill. I tend towards leaving it as is, or perhaps use an "optional plural" form like "Selecteer bestand(en)"

Reply:
I changed the label simply based on multiple. I don't think we need to have plural forms based on the length of the selection.